src/librustc_borrowck/borrowck/check_loans.rs

```diff
@@ -659,7 +659,7 @@ impl<'a, 'tcx> CheckLoanCtxt<'a, 'tcx> {
         debug!("check_if_path_is_moved(id={:?}, use_kind={:?}, lp={:?})",
                id, use_kind, lp);
 
-        // FIXME (22079): if you find yourself tempted to cut and paste
```

Comment:
This points to a PR, not an issue. It's fine to remove as you're doing, just pointing it out.

Reply:
I know, it's just that the FIXME is general and the number makes it seem like it is waiting on problem to be fixed.